docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -269,19 +269,30 @@ content: |
 ---
 id: MLH outro electronic signature
 question: |
-  Electronic Signatures
+  Add your electronic signature?
 subquestion: |
-  This tool can put "/s/ ${users[0].name.full()}" where you would sign your name so you do not have to sign the forms by hand. The court will accept this as your signature. 
-
-  If you plan to e-file, you may want to add your electronic signature, otherwise you will need to print your forms, sign them by hand, and then scan them.
-
-  Would you like to add your electronic signature to your forms? 
-  ${ collapse_template(efiling_info_template) }
+  You can sign Michigan court forms electronically or by hand. The court will accept either type of signature.
 fields:
-  - no label: MLH_esign
+  - Would you like to add your electronic signature to your forms now?: MLH_esign
     datatype: yesnoradio
-continue button field: MLH_outro_electronic_signature
+    label above field: True
+    help: |
+      If you choose **Yes**, the forms will be signed "*/s/ ${users[0].name.full()}*" when you download them. 
+under: |
+  ${ collapse_template(signature_info_template) }
+---
+template: signature_info_template
+subject: |
+  I plan to **e-file**. What should I choose?
+content: |
+  If you **plan to e-file**, you may want to add your electronic signature. Otherwise you will need to print your forms, sign them by hand, and then scan them.
 
+  To find out if your court has e-filing, read the [What is E-Filing?](https://michiganlegalhelp.org/resources/mifile/what-e-filing) article.
+  The article tells you how to find out which courts have e-filing for which case types.
+  
+  ${ MLH_case_type_language }
```

Comment:
I saw this is blank by default; do you have an example of what it look like if it wasn't blank?

Reply:
@normon66 correct me if I'm wrong, but I'm imagining it would be something like: "The case type for name change is NC."

Reply:
@ekressmiller - The wording listed in the Google sheet for name change is just "The case type code is NC.", but I like your extra specification better! It also goes on to talk about subsequent filings and such.